### PR TITLE
ci: clean-up hard drive more often

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Review by aherrmann-da — APPROVED** [review]
LGTM

File: ci/build.yml [hunk 1]
```diff
@@ -148,6 +150,7 @@ jobs:
     - template: report-start.yml
     - checkout: self
     - bash: ci/dev-env-install.sh
+    - temaplte: clean-up.yml
```
Comment (SamirTalwar-DA) [inline ci/build.yml:153]: Typo.

File: ci/build.yml [hunk 2]
```diff
@@ -207,6 +210,7 @@ jobs:
   steps:
     - template: report-start.yml
     - checkout: self
+    - temaplte: clean-up.yml
```
Comment (SamirTalwar-DA) [inline ci/build.yml:213]: Likewise.

File: ci/build.yml [hunk 3]
```diff
@@ -223,6 +227,7 @@ jobs:
...
```
Comment (SamirTalwar-DA) [inline ci/build.yml:230]: And here.

File: ci/build.yml [hunk 4]
```diff
@@ -242,6 +247,7 @@ jobs:
     - template: report-start.yml
     - template: clear-shared-segments-macos.yml
     - checkout: self
+    - temaplte: clean-up.yml
```
Comment (SamirTalwar-DA) [inline ci/build.yml:250]: Same.

File: ci/cron/daily-compat.yml [hunk 5]
```diff
@@ -30,6 +30,7 @@ jobs:
       demands: assignment -equals default
     steps:
       - checkout: self
+      - temaplte: ../clean-up.yml
```
Comment (SamirTalwar-DA) [inline ci/cron/daily-compat.yml:33]: Two more.

File: ci/cron/daily-compat.yml [hunk 6]
```diff
@@ -49,6 +50,7 @@ jobs:
       - checkout: self
       - ${{ if eq(variables['pool'], 'macos-pool') }}:
         - template: ../clear-shared-segments-macos.yml
+      - temaplte: ../clean-up.yml
```
Comment (SamirTalwar-DA) [inline ci/cron/daily-compat.yml:53]: One more.

**Review by SamirTalwar-DA — APPROVED** [review]

File: ci/build.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/build.yml]: Thanks! They were all wrong. I guess I should double-check before copy/pasting. All fixed up now. I think.

**Pull request merged**